subprojects/testing-native/src/main/java/org/gradle/nativeplatform/test/cpp/internal/DefaultCppTestSuite.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.nativeplatform.test.cpp.internal;
+
+import org.gradle.api.artifacts.ConfigurationContainer;
+import org.gradle.api.internal.file.FileOperations;
+import org.gradle.api.model.ObjectFactory;
+import org.gradle.language.cpp.internal.DefaultCppApplication;
+import org.gradle.nativeplatform.test.cpp.CppTestSuite;
+
+import javax.inject.Inject;
+
+public class DefaultCppTestSuite extends DefaultCppApplication implements CppTestSuite {
```

Comment:
should be `extends DefaultCppComponent` rather than `extends DefaultCppApplication`.

Reply:
done